spec/models/case/sar/offender_spec.rb

```diff
@@ -328,9 +328,36 @@
         expect(kase.errors[:third_party_relationship]).to eq ["cannot be blank"]
       end
 
-      it "does not validates presence of third party relationship when recipient is not third party" do
+      it "does not validate presence of third party relationship when recipient is not third party" do
         kase = build_stubbed :offender_sar_case, third_party: false, third_party_relationship: "",
-                                                 recipient: "subject_recipient"
+                                                 third_party_email: "", third_party_company_name: "", recipient: "subject_recipient"
+        expect(kase).to be_valid
+      end
+    end
+
+    describe "third party email" do
+      it "validates email address when there is an email address entered" do
+        kase = build_stubbed :offender_sar_case, third_party: true, third_party_relationship: "Solicitor",
+                                                 third_party_email: "email", third_party_company_name: "",
+                                                 recipient: "third_party_recipient"
+
+        expect(kase).not_to be_valid
+        expect(kase.errors[:third_party_email].to_sentence)
+      end
+
+      it "does not validate email address when there is no email address entered" do
+        kase = build_stubbed :offender_sar_case, third_party: true, third_party_relationship: "Solicitor",
+                                                 third_party_email: "", third_party_company_name: "ABC LLP",
+                                                 recipient: "third_party_recipient"
+
+        expect(kase).to be_valid
+      end
+
+      it "validates email address when there is an email address entered" do
```

Comment:
This test has the same name as a previous one. Also the entered email address is invalid, but is passing.